ui/public/i18n/language.json

```diff
@@ -24,7 +24,35 @@
     "title": "Settings",
     "configure_instance": "Configure {instance}",
     "save": "Save",
-    "test_filed": "Test field"
+    "roundcubemail_fqdn": "Roundcubemail hostname (FQDN)",
```

Comment:
```suggestion
    "roundcubemail_fqdn": "Roundcube hostname (FQDN)",
```

Roundcubemail is the name of the GIT repo, but the name of the product is Roundcube